src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -6838,6 +6838,17 @@ components:
           type: string
         subfilter:
           $ref: '#/components/schemas/Filter'
+        group_by_count_attribute:
```

Comment:
Seems like tanagra may have made some changes to this while we were developing, so I updated the yaml

Reply:
OK. I've been tracking the changes made upstream to see if anything applicable to our work has happened. So far most changes I've seen are in the web UI code, or in the json configuration files.